.github/workflows/e2e-tests-ondemand.yaml

```diff
@@ -67,3 +67,21 @@ jobs:
           tenant2-name: ${{ secrets.TENANT2_NAME }}
           tenant2-apitoken: ${{ secrets.TENANT2_APITOKEN }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
+
+
+  notify-failure:
+    name: Notify failure in Slack
+    environment: E2E ondemand
+    needs: [ "run-matrix" ]
+    if: ${{ failure() || cancelled() }}
+    runs-on:
+      - self-hosted
+      - operator-e2e
```

Comment:
does this need to run on our github runners?

Reply:
@chrismuellner good catch, I've copy/pasted it from https://github.com/dynatrace/dynatrace-operator/blob/bf7b156866c93042a5bf32838136f10dec143162/.github/workflows/e2e-tests.yaml#L70-L72

cc @albertogdd

Reply:
Good point, it shouldn't be needed

Reply:
but which runner is correct ?
```
runs-on: ubuntu-latest
```
?

Reply:
I think you don't even need to specify it if you don't need anything specific

Reply:
let's use the same runner as other places, which should be `ubuntu-latest`

Reply:
Linux always
<img src="https://media0.giphy.com/media/l0Hlw1wlzvxTvxiZG/giphy-downsized-medium.gif"/>

Reply:
fixed in 4204ad8c